identity/webapp/pages/registration.tsx

```diff
@@ -49,18 +49,20 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
 
     try {
       const token = decodeToken(sessionToken);
+      console.log({ token });
```

Comment:
Aren't these tokens potentially sensitive? Do we want to be logging them?

Reply:
Okay. If we do think it's okay to log them, let's log them with a bit more information – e.g.

```suggestion
      console.log(`Decoded post-registration token as $token`);
```

Reply:
Can do – but this change is only due to be ephemeral and using the shorthand object property name will make it clear that this is the `token` that's being logged